accounts/utils.py

```diff
@@ -0,0 +1,21 @@
+from .backend import OpenStax
+from django.conf import settings
```

Comment:
not used - but I'm not opposed to IMPORT_PIPELINE living in settings, since it's possible it could need to change